pom.xml

```diff
@@ -27,11 +27,11 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <!-- Primary components -->
         <java.version>1.8</java.version>
-        <scala.version>2.10</scala.version>
-        <flink.version>1.3.2</flink.version>
+        <scala.version>2.11</scala.version>
+        <flink.version>1.9.0</flink.version>
 
         <!-- Secondary components -->
-        <async.client.version>2.0.39</async.client.version>
+        <async.client.version>2.10.4</async.client.version>
         <logback.version>1.2.3</logback.version>
         <guava.version>23.0</guava.version>
         <typesafe.config.version>1.3.3</typesafe.config.version>
```

Comment:
Probably we should remove this dependency or put it in test scope, cause we use it only in tests.